dotnet/README.md

```diff
@@ -0,0 +1,51 @@
+## Getting Started with SFTP using C#(.NET)
+
+#Project Description
+
+The project contains two solutions:
+- SFTPClient: This is a simple console application which depicts how we can use our SFTP service to perform differnet file operations.
+- SFPTService: This is a simple class library project that conatains the contains code to connect, upload , dowload and delete files.
+
+#Project Dependencies
+
+- Install SSH.NET library from Nuget Pacakage.
+
+# Getting Started 
+
+1. First we need to instaill SSH.NET pacakage. We can either do this directly from the Pacakage manager or using the below command:
+
+	dotnet add package SSH.NET --version 2020.0.0-beta1
+
+2. Next, we need to define our SFTP Server configurations such as host name , port number , username and password.
+
+	```cs
+			public class SftpConfig
+			{
+				public string Host { get; set; }
+				public int Port { get; set; }
+				public string UserName { get; set; }
+				public string Password { get; set; }
+			}
+	```
+
+3. The next step is to simply call our SFTP service.Any operation performed inside the SFTP service would have :
```

Comment:
@paras-lalchandani  Some typos: `The next step is to simply call our SFTP service. Any operation performed inside the SFTP service would have to:`